src/utils.js

```diff
@@ -1,4 +1,12 @@
 /*
+ * Implementation of String.endsWith, which errors in node.
+ *
+ */
+export function endsWith(string, suffix) {
+  return string.indexOf(suffix, string.length - suffix.length) !== -1;
```

Comment:
Damn, good catch. I actually was thinking "I think I left out a test for something" but couldn't remember what.